### DISCUSSION TITLE
AV1 encoder quits halfway without reason with a completed message

### DISCUSSION THREAD

**Discussion by Neo166** [mafiosnik777/enhancr#27] [General]
Hi Team,
First of all, I am very impressed by enhance. It's really good. However, I am facing problems with its reliability. AV1 encodes more than 70% of the times do not finish. The frames encoded will encode till just over half the frames then it will end saying the  encoding is complete. It is very frustrating since the CPU based AV1 encoding takes almost 90 mins to 120 mins and suddenly after an hour it will just finish abruptly. I do not face this issue if I use the AV1 NVENC hardware encoders. This is not a cache problem too since I always maintained 100 GB of free space.

This is the settings what I use:
-c:v libsvtav1 -qp 50 -preset 7 -svtav1-params tune=0:enable-tf=0:enable-overlays=1:enable-qm=1 -pix_fmt yuv420p10le

Note: qp is 50 since its Anime and I don't see much quality degradation.

Regards

**Comment by mafiosnik777** [comment]
Known issue, can't fix since Vapoursynth is a black box without error messages and just crashes sometimes.
